onmt/modules/Generator.lua

```diff
@@ -9,28 +10,67 @@ local Generator, parent = torch.class('onmt.Generator', 'onmt.Network')
 -- for back compatibility - still declare FeaturesGenerator - but no need to define it
 torch.class('onmt.FeaturesGenerator', 'onmt.Generator')
 
+local options = {
+  {
+    '-importance_sampling', false,
```

Comment:
for end-user, it is a generator option - the name is not very intuitive since it initially meant "importance sampling of target vocabulary" - while I am not doing sampling any more. an alternative name is welcome